### PR TITLE
STL in C++ course(bangla) added 

### PR DESCRIPTION
**Pull request by khalid586**
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
 Shikkhangon BD
 * [Object Oriented C++ \| Bangla Tutorial](https://www.youtube.com/playlist?list=PLy7uM3PHzMF1hnqhFGE4_A8qTUfFmZ_3y) - 
 Online School
+* [Standard Template Library in C++](https://youtube.com/playlist?list=PLgLCjVh3O6Sgux985GYG22xkFt9z9Sq0_) - LoveExtendsCode
```
Comment (eshellman) [inline courses/free-courses-bn.md:76]: Don't invent titles! Use the title given by the resource.

**Comment by khalid586** [comment]
@eshellman done!

**Comment by khalid586** [comment]
@eshellman  please review the changes and merge if everything's okay. Thank you.

**Comment by eshellman** [comment]
didd you mean to add the competitive programming link to this PR? Either remove it or explain it.

**Comment by khalid586** [comment]
> didd you mean to add the competitive programming link to this PR? Either remove it or explain it.

I added them by mistake. Removed them. I hope you will be able to merge now.

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -73,6 +73,7 @@
 Shikkhangon BD
 * [Object Oriented C++ \| Bangla Tutorial](https://www.youtube.com/playlist?list=PLy7uM3PHzMF1hnqhFGE4_A8qTUfFmZ_3y) - 
 Online School
+* [C++ STL (Bangla)](https://youtube.com/playlist?list=PLgLCjVh3O6Sgux985GYG22xkFt9z9Sq0_) - LoveExtendsCode
```
Comment (eshellman) [inline courses/free-courses-bn.md:76]: please put lists in alphabetical order

Comment (khalid586) [reply courses/free-courses-bn.md:73]: @eshellman alphabetically sorted.

**Comment by khalid586** [comment]
@eshellman I have sorted alphabetically. Have a look at it. Thanks.

**Comment by eshellman** [comment]
thanks

**Pull request merged**